src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -155,6 +155,40 @@ private static Response ProcessResponse(Response response)
                 restException.MoreInfo
             );
         }
+
+        public static void ValidateSslCertificate()
+        {
+            ValidateSslCertificate(DefaultClient());
+        }
+
+        public static void ValidateSslCertificate(HttpClient testClient)
```

Comment:
is it better to make this an optional param, to save the trouble of overloading the method?

Reply:
I went with overloading for the sake of code completion--figured it's simpler to see a version with no parameter than trying to figure out why the default is null.

I added some comments to these methods in the next commit that I just pushed.